404.html

```diff
@@ -0,0 +1,19 @@
+---
+layout: default
+---
+
+<div class="row">
+    <div class="container my-5 py-5">
+        <div class="offset-lg-3 col-lg-7 mt-md-4">
+            <img alt="" width="40" height="40" src="assets/alert.svg">
+            <h1 class="h2 mb-3 pt-4 mt-3">Something's missing!</h1>
+        </div>
+        <div class="offset-lg-3 col-lg-6 mb-md-4">
+            <p class="mb-3 pb-3 text-info">The page you’re looking for doesn’t exist or was removed. We suggest you go
+                back home.</p>
+            <div class="d-flex flex-column flex-lg-row mb-5 pb-5">
+                <a class="btn btn-primary" href="/">Go back home</a>
+            </div>
+        </div>
+    </div>
+</div>
```

Comment:
The ordering of elements here is slightly off.

We always want `.row > .col-x` e.g. columns are direct descendants of rows.

Here `.container` is sitting in the middle, it should be on the outside wrapping the `.row`.

https://getbootstrap.com/docs/5.2/layout/grid/

Reply:
I'm noticing this by going back and forth to a found / not found page, and seeing the widths are different.

E.g. https://deploy-preview-127--compilerla.netlify.app/jobs (top) vs. https://deploy-preview-127--compilerla.netlify.app/nope (bottom)

<img width="950" alt="image" src="https://github.com/compilerla/compiler.la/assets/1783439/ad4d5470-4ecc-48ad-bfc9-fb8758a43688">

Reply:
Yeah, I think it would be better if they had the same offset, but the design calls for them to be different. @segacy1 do you have thoughts on this?

![image](https://github.com/compilerla/compiler.la/assets/25497886/f54e96c0-3454-44cf-b504-4c5b2dd78c7f)

![image](https://github.com/compilerla/compiler.la/assets/25497886/6113166b-479b-4473-9a0b-a54f14836a54)

Reply:
Let's match it to jobs! Will update in figma now

Reply:
> The ordering of elements here is slightly off.

There's a [`.container-fluid`](https://github.com/compilerla/compiler.la/blob/main/_layouts/default.html#L51) from the default layout that wraps the `.row`.

> We always want `.row > .col-x` e.g. columns are direct descendants of rows.

Yeah, I also noticed that in [lines 6-7](https://github.com/compilerla/compiler.la/pull/127/files#diff-128cc35baaf40bac8ae4d2932ca6e009311e8279f03f258e9cc83966f1d5b32eR6-R7) we don't have a `.row` in between the `.container` and `.col`. I can fix that. I was going off what's in [`index.html`](https://github.com/compilerla/compiler.la/blob/a7abe22129f32bf7311b9d117a24c2d216a4f30d/index.html#L7-L8), which is also missing the `.row`.

Reply:
@segacy1 if possible, every page should have the same offset / width in the content area. It is quite jarring going between pages when that changes. (While also noting that some pages obviously need more space, e.g. for a hero image or something)